nifi-assembly/NOTICE

```diff
@@ -289,6 +289,11 @@ The following binary components are provided under the Apache Software License v
       Apache Tika Core
       Copyright 2007-2015 The Apache Software Foundation
 
+  (ASLv2) Apache Tika
+    The following NOTICE information applies:
```

Comment:
The Tika notice should be reflected here: https://github.com/apache/tika/blob/master/NOTICE.txt